src/utils/barcode/barcode-scan.js

```diff
@@ -83,15 +83,16 @@ const csvToJson = (csvData) => {
  * @returns {object} returnedData - JSON object containing scanned barcode data
  */
 const getDriversLicenseData = async (file) => {
-  const image = await readImageFile(file);
-  const decoded = await decodeBarcode(image);
-  let returnedData = '';
-  if (decoded) {
-    const jsonData = csvToJson(decoded);
-    returnedData = jsonData;
-  } else {
-    returnedData = new Error('Unable to decode file');
+  let returnedData = {};
```

Comment:
What are these changes?

Reply:
This is to make sure that the scanner takes in only image files.

Reply:
Relegating these fixes to PR #264.